src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -556,7 +555,7 @@ public class @class
     {
     }
 
-    public static void Add(object t)
+    public static void Add(object @class)
```

Comment:
❕ This change makes it impossible to tell if the test is using the correct value for the renamed identifier. You should rename `class @class` at the same time so it's clear that the verbatim identifier comes from the parameter name, not the type name.